dicom-dimse-service/src/main/resources/application.properties

```diff
@@ -35,12 +35,11 @@ dimse.nats.traceConnection=false
 dimse.nats.reply.timeoutSeconds=60
 dimse.nats.chunk.size=262144
 
-dimse.called.host=localhost
-dimse.called.port=${dimse.ingestion.port}
-dimse.called.aet=${dimse.ingestion.aet}
-dimse.called.device=${dimse.ingestion.device}
-dimse.calling.aet=LOCAL
+dimse.target.host=localhost
+dimse.target.port=${dimse.port}
+dimse.target.ae=${dimse.ae}
+dimse.target.device=${dimse.device}
 
 dimse.config.path=${CONFIG_PATH:/etc/config/dimse}
 
-dimse.proxy.actor=SERVER
+dimse.actor=SCP
```

Comment:
This should be SCU.  The proxy will act as a SCU to the ingestion code.  This one is a little tricky conceptually as the ingestion is embedded directly.